### PR TITLE
Added  NMCBit Prop BTC &  Updated hash rates 

### PR DESCRIPTION
**Pull request by Locky7**
Added  NMCBit Prop BTC+cfg

Added Eligius hash rate

Added Ozco, Bloodys, Yourbtc block verification

Public Domain

### CODE REVIEW

File: pools.cfg [hunk 1]
```diff
@@ -117,9 +117,14 @@ url: http://btcworld.de/dashboard
...
-api_method: json
-api_key: solved
+#api_address: http://api.triplemining.com/json/stats
+#api_method: json
+#api_key: solved
+api_address: https://www.triplemining.com/stats
+api_method: re
+api_key: <!-- 1000 -->([0-9]+( [0-9]+)*)</td>
+api_key_ghashrate: ([0-9.]+) GH/s
+api_strip:' '
```
Comment (paraipan) [inline pools.cfg]: hey, sup with the json, it work fine here, why the change ?

Comment (Locky7) [inline pools.cfg]: json and scraping the stats page provide the same share count, however, I can't get hash rate via json

Figured I wasn't taking anything away, but adding functionality...

Comment (paraipan) [inline pools.cfg]: ok agree but hammering the webpage it's not a desired effect, i see that json is working on triple from my location. We're quite a few hoppers these days so using json is needed ;)

Comment (Locky7) [inline pools.cfg]: it's all good, should we keep it or should I mod the pull?  I find seeing the hash rate important for mine_lp and DDoS...but wasn't aware of the  costs.

Comment (paraipan) [inline pools.cfg]: leave the scrape in there, as backup, and comment it out.  You can do the changes, commit them and make a git push. The new code is added automatically to the pull req
